atomic_reactor/cli/parser.py

```diff
@@ -173,6 +173,12 @@ def _add_common_task_args(task_parser: argparse.ArgumentParser) -> None:
         required=True,
         help="PipelineRun name to reference current PipelineRun"
     )
+    task_parser.add_argument(
+        "--task-result",
+        metavar="FILE",
+        default=None,
+        help="file to write task result",
```

Comment:
That doesn't sound quite right. We don't use it for task results but only for the error message. Which can't even be a task result, because failed tasks don't produce results.

How about something like `--write-error`?

Reply:
sure